### PR TITLE
[Behat] Fix base channel currency step

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | |
| License         | MIT |

I open this PR once more time to avoid merge problems on ``currency-party``.

### CODE REVIEW

**Comment by michalmarcinkowski** [comment]
@Zales0123 rebase needed...

**Comment by michalmarcinkowski** [comment]
@Zales0123 the build is failing, looks like we need some fixes.

**Comment by Zales0123** [comment]
@michalmarcinkowski it should work now.

File: src/Sylius/Component/Core/Test/Services/DefaultChannelFactory.php [hunk 1]
```diff
@@ -118,16 +118,20 @@ public function create($code = null, $name = null)
...
 
     /**
+     * @param string|null $currencyCode
+     *
      * @return CurrencyInterface
      */
-    private function provideCurrency()
+    private function provideCurrency($currencyCode = null)
     {
+        $currencyCode = (null === $currencyCode) ? self::DEFAULT_CHANNEL_CURRENCY : $currencyCode;
```
Comment (lchrusciel) [inline src/Sylius/Component/Core/Test/Services/DefaultChannelFactory.php]: `$currencyCode = $currencyCode ? : self::DEFAULT_CHANNEL_CURRENCY`

**Comment by michalmarcinkowski** [comment]
👍

**Comment by lchrusciel** [comment]
🎉

**Pull request merged**